packages/venia-concept/src/components/App/app.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import { bool, func, shape, string } from 'prop-types';
 
 import classify from 'src/classify';
-import Main from 'src/components/Main';
+import Page from 'src/components/Page';
```

Comment:
We've had more than one concept named "Page", so I'm concerned that this rename will cause confusion. Can you please explain why this rename relates to accessibility?

Reply:
In general, the main change was to replace root component from `div` to `main` inside Page component. 
`<main>` element should not include global `<header>` and `<footer>`, because the purpose of the `<main>` tag is to include the content that is unique for each particular page. After this changes the name of the component was no longer in compliance with the code in file. Do you have ideas what will be a better name for the file.